src/migrations/1738599982159-initial-schema.ts

```diff
@@ -580,27 +580,6 @@ export class InitialSchema1738599982159 implements MigrationInterface {
     }
 
     public async down(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query(`
-            ALTER TABLE "category_key_info" DROP CONSTRAINT "FK_ec0b41bafd5605fff51fc0c8e47"
-        `);
-        await queryRunner.query(`
-            ALTER TABLE "category_info" DROP CONSTRAINT "FK_68028565126809c1e925e6f9334"
-        `);
-        await queryRunner.query(`
-            ALTER TABLE "hierarchy" DROP CONSTRAINT "FK_6ca6a866371cdd67a638df8a74c"
-        `);
-        await queryRunner.query(`
-            ALTER TABLE "hierarchy" DROP CONSTRAINT "FK_aaba494e02eaa91111d549e5763"
-        `);
-        await queryRunner.query(`
-            ALTER TABLE "reference_data_info" DROP CONSTRAINT "FK_f671fde9c769286ba971485ba09"
-        `);
-        await queryRunner.query(`
-            ALTER TABLE "reference_data" DROP CONSTRAINT "FK_dd4ff535904e339641b0b0d52c2"
-        `);
-        await queryRunner.query(`
-            ALTER TABLE "category_key" DROP CONSTRAINT "FK_087b36846d67092609821a62756"
-        `);
```

Comment:
Please can we avoid editing existing schema files as they've already been run and we'll end up with out-of-sync DBs... Can we add all the required changes to the new migration file? 

Failing that, let's just replace the initial schema completely with a fresh single migration.

Reply:
Restored the initial migration file